.env.sample

```diff
@@ -1,11 +1,19 @@
+# Pocket RPC Configuration
 POKT_RPC_FULL_HOST=
-HTTP_SERVER_PORT=8080
 POKT_RPC_TIMEOUT=5s
+
+# Gateway Deployment Configuration
+HTTP_SERVER_PORT=8080
 ENVIRONMENT_STAGE=development
-POKT_APPLICATIONS_ENCRYPTION_KEY=
+EMIT_SERVICE_URL_PROM_METRICS=false
+
+# Pocket Business Logic
+CHAIN_NETWORK=morse_mainnet
 SESSION_CACHE_TTL=75m
-DB_CONNECTION_URL=postgres://myuser:mypassword@postgres:5432/postgres?sslmode=disable
-API_KEY=
 ALTRUIST_REQUEST_TIMEOUT=10s
-EMIT_SERVICE_URL_PROM_METRICS=false
-CHAIN_NETWORK=morse_mainnet
\ No newline at end of file
+API_KEY=
+
+# App Stake Management
+DB_CONNECTION_URL=postgres://myuser:mypassword@postgres:5432/postgres?sslmode=disable
```

Comment:
The default port for the DB in their example is 5433 so I think we should change either the example docker compose or example .env to match.